alcs-frontend/src/app/features/board/dialogs/amendment/create/create-amendment-dialog.component.ts

```diff
@@ -6,11 +6,11 @@ import { debounceTime, distinctUntilChanged, Observable, startWith, switchMap }
 import { ApplicationAmendmentCreateDto } from '../../../../../services/application/application-amendment/application-amendment.dto';
 import { ApplicationAmendmentService } from '../../../../../services/application/application-amendment/application-amendment.service';
 import { ApplicationRegionDto, ApplicationTypeDto } from '../../../../../services/application/application-code.dto';
+import { ApplicationDecisionService } from '../../../../../services/application/application-decision/application-decision.service';
 import { ApplicationLocalGovernmentDto } from '../../../../../services/application/application-local-government/application-local-government.dto';
 import { ApplicationLocalGovernmentService } from '../../../../../services/application/application-local-government/application-local-government.service';
 import { ApplicationDto } from '../../../../../services/application/application.dto';
 import { ApplicationService } from '../../../../../services/application/application.service';
-import { CardService } from '../../../../../services/card/card.service';
 import { ToastService } from '../../../../../services/toast/toast.service';
 
 @Component({
```

Comment:
are we applying onDestroy for all forms that use subscribe?

Reply:
yes we should! Ill go through and fix the other dialogs.

Reply:
Done!